src/app/_components/MobileNavigation.tsx

```diff
@@ -19,28 +19,40 @@ import { PATHS } from '@/constants/paths'
 const getInvolvedItems = [PATHS.EVENTS, PATHS.GRANTS]
 const communityItems = [PATHS.ECOSYSTEM, PATHS.GOVERNANCE]
 
-const linkTouchTarget = {
-  class: 'inline-block px-5 py-2.5',
-  offsetClass: '-ml-5',
+type IconButtonProps = {
+  icon: IconProps['component']
+  label: string
+  onClick: React.ComponentPropsWithoutRef<'button'>['onClick']
 }
 
-type MobileLinkProps = {
+function IconButton({ icon: IconComponent, label, onClick }: IconButtonProps) {
+  return (
+    <button
+      aria-label={label}
+      className="grid size-12 place-items-center rounded-lg border border-brand-300 text-brand-300 focus:outline-2 focus:outline-brand-100"
+      onClick={onClick}
+    >
+      <Icon component={IconComponent} />
+    </button>
+  )
+}
+
+type LinkItemProps = {
   label: string
   path: Route
   nested?: boolean
   setOpen: Dispatch<SetStateAction<boolean>>
 }
 
-function MobileLink({ label, path, nested, setOpen }: MobileLinkProps) {
+function LinkItem({ label, path, nested, setOpen }: LinkItemProps) {
   return (
-    <li
-      className={clsx(
-        nested ? 'border-l border-brand-400' : linkTouchTarget.offsetClass,
-      )}
-    >
+    <li className={clsx(nested && 'ml-6')}>
       <Link
         href={path}
-        className={clsx(linkBaseStyles, linkTouchTarget.class)}
+        className={clsx(
+          linkBaseStyles,
+          "relative before:absolute before:inset-0 before:-m-2 before:content-[''] after:absolute after:inset-0 after:-m-2 after:content-['']",
+        )}
```

Comment:
@CharlyMartin ~ What do you think about this solution for touch targets?

Reply:
@mirhamasala I like it very much 👏 Let's find a way to add it to our config!

I increased the negative margins of the before element to be as close to 48px as possible. I also removed the after pseudo-class (only one should be necessary)